Demand block hashes from branch point (#905)
planetarium/libplanet: CHANGES.md

To be released.

>  -  Fixed a bug that did not properly receive block hashes after the chain has reorged.

Reviewer: ```suggestion
 -  Fixed a bug that had not properly received block hashes after the chain had reorged.
```
